### DISCUSSION TITLE
加个对局counter建议怎么样

### DISCUSSION THREAD

**Discussion by zhangzhengyi12** [cangzhang/ChampR#156] [General]
挺喜欢这个软件的，很实用，在和朋友玩的时候都感觉加入一个对局Counter提示的功能会更好。
比如说对面选择英雄A之后，我可以指定这个英雄为我的对位，然后自动列举克制和反克制的英雄列表，不知道你感觉怎么样。
如果可以的话，我可以贡献这部分的代码

**Comment by zhangzhengyi12** [comment]
另外我觉得比较重要的功能是强制桌面应用层级最高，ｗｅｇａｍｅ有这个功能，天赋选择弹窗很容易被其他应用遮挡，经常需要手动最小化来选择天赋，当然ｗｉｎｄｏｗｓ的原生开发这块我也不是很懂，ｅｌｅｃｔｒｏｎ应该也可以做到吧

**Comment by cangzhang** [comment]
@zhangzhengyi12 天赋弹窗是有置顶按钮的，在英雄头像的左侧，有一个 pin 的按钮

**Comment by whj768** [comment]
是的作者能否加入康特英雄 英雄层级这样的 @cangzhang 大佬谢谢
